experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -35,27 +47,30 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
         @warn("Running with ρe_int in coupled mode is not tested yet.", maxlog = 1)
     end
 
-    # define shorter references for long variable Interfacer.names to increase readability
+    # define shorter references for long variable Interfacer.names to increase readability, and set to zero
```

Comment:
oops - I think I changed "names" here to "Interfacer.names" when I was updating our import/using syntax... would you mind removing the "Interfacer."?